src/views/CreateNewCardCustomPicker.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+	<div class="modal-scroller">
+		<div v-if="!creating && !created" id="modal-inner" :class="{ 'icon-loading': loading }">
+			<h3>{{ t('deck', 'Create a new card') }}</h3>
+
+			<input v-model="card.title"
+				type="text"
+				:placeholder="t('deck', 'Card title')"
+				:disabled="loading">
+
+			<div class="row">
+				<div class="col">
+					<NcMultiselect v-model="selectedBoard"
+						:placeholder="t('deck', 'Select a board')"
+						:options="boards"
+						:disabled="loading"
+						label="title"
+						class="multiselect-board"
+						@select="fetchBoardDetails">
+						<template slot="singleLabel" slot-scope="props">
+							<span>
+								<span :style="{ 'backgroundColor': '#' + props.option.color }" class="board-bullet" />
+								<span>{{ props.option.title }}</span>
+							</span>
+						</template>
+						<template slot="option" slot-scope="props">
+							<span>
+								<span :style="{ 'backgroundColor': '#' + props.option.color }" class="board-bullet" />
+								<span>{{ props.option.title }}</span>
+							</span>
+						</template>
+					</NcMultiselect>
+				</div>
+
+				<div class="col">
+					<NcMultiselect v-model="selectedStack"
+						:placeholder="t('deck', 'Select a list')"
+						:options="stacksFromBoard"
+						:max-height="100"
+						:disabled="loading || !selectedBoard"
+						class="multiselect-list"
+						label="title" />
+				</div>
+			</div>
+
+			<TagSelector :card="card"
+				:labels="labels"
+				:disabled="loading || !selectedBoard"
+				@select="onSelectLabel"
+				@remove="onRemoveLabel"
+				@newtag="addLabelToBoardAndCard" />
+
+			<div>
+				<AssignmentSelector :card="card"
+					:assignables="assignables"
+					:can-edit="!loading && selectedBoard"
+					@select="onSelectUser"
+					@remove="onRemoveUser" />
+			</div>
+
+			<div class="section-details">
+				<DueDateSelector :card="card" :can-edit="canEdit" @change="updateCardDue" />
+			</div>
+
+			<textarea v-model="pendingDescription" :disabled="loading || !selectedStack" :placeholder="t('deck', 'Card description')" />
```

Comment:
This should switch to use the `Description` component so we also have the rich text editor